3D/Web/SRC/counter.js

```diff
@@ -0,0 +1,9 @@
+export function setupCounter(element) {
+  let counter = 0
+  const setCounter = (count) => {
```

Comment:
This doesn't respect state management. Could this call `useState()` ?

Reply:
I do not think so.